Experimental/DedicatedGameServer/Assets/Editor/UI.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I think the whole Editor folder can be removed, no? It's empty for me.

Reply:
Nice catch, removed!